### PR TITLE
feat: new tones for Releases

### PR DESCRIPTION
**Pull request by mariuslundgard**
This change introduces 2 new tones: `neutral` and `suggest`.

These tones work for cards:

```tsx
<Card tone="neutral" />
<Card tone="suggest" />
```

And other elements that take the `tone` property:
```tsx
<Badge tone="neutral" />
<Button tone="suggest" />
```

### CODE REVIEW

File: src/theme/build/resolveColorTokens.ts [hunk 1]
```diff
@@ -162,44 +142,22 @@ function resolveButtonStateColorTokens(
...
-        ...spec.badge?.['*'],
-        ...spec.badge?.caution,
-      },
-      critical: {
-        _hue: inputTokens?.base?.critical?._hue,
-        ...spec.badge?.['*'],
-        ...spec.badge?.critical,
-      },
-    },
+    avatar: COLOR_HUES.reduce((acc, hue) => {
```
Comment (bjoerge) [inline src/theme/build/resolveColorTokens.ts:145]: nice simplification! 😍

File: src/theme/defaults/colorTokens.ts [hunk 2]
```diff
@@ -74,6 +74,7 @@ export const defaultColorTokens: ThemeColorTokens = {
       },
     },
     'primary': {_hue: 'blue'},
+    'potential': {_hue: 'purple'},
```
Comment (bjoerge) [inline src/theme/defaults/colorTokens.ts:77]: not a blocker: should we document somewhere what we mean by "potential"? or is it an established term?

**Review by bjoerge — DISMISSED** [review]
Nice, LGTM!

File: src/theme/defaults/colorTokens.ts (hunk 2 above)
Comment (mariuslundgard) [reply src/theme/defaults/colorTokens.ts:77]: Yes, good point 👍

**Review by bjoerge — DISMISSED** [review]

**Review by bjoerge — APPROVED** [review]

**Pull request merged**